scripts/test-projects-workspace/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "test-projects-workspace"
+description = "Build project workspace with `forc-pkg`"
+version = "0.1.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+publish = false
+
+[dependencies]
+forc-pkg = "0.32.2"
+forc-util = "0.32.2"
+#forc-fmt = "0.32.2" todo do we need this
+walkdir = "2.3.2"
+glob = "0.3.1"
+
+termcolor = "1.1"
+tokio = { version = "1.21.2", features = ["process", "macros", "rt", "rt-multi-thread"] }
+futures-util ="0.3.24"
```

Comment:
AFAIK we do not need `tokio` and `fucures-util` as you only run one build now.

Reply:
I added it because I was testing fmt. In the end, we will see if it will be needed or not. Perhaps none of these changes will be necessary.